packages/cli/src/dbt/models.ts

```diff
@@ -212,9 +212,33 @@ export const findAndUpdateModelYaml = async ({
         const newColumns = generatedModel.columns.filter(
             (c) => !existingColumnNames.includes(c.name),
         );
+        const deletedColumnNames = existingColumnNames.filter(
+            (c) => !generatedModel.columns.map((gc) => gc.name).includes(c),
+        );
+        let updatedColumns = [...existingColumnsUpdated, ...newColumns];
+        if (deletedColumnNames.length > 0 && process.env.CI !== 'true') {
+            spinner.stop();
+            const answers = await inquirer.prompt([
+                {
+                    type: 'confirm',
+                    name: 'isConfirm',
+                    message: `
+These columns in your .yml file no longer exist in your warehouse:
```

Comment:
Can we have the file and model name in this message instead of just `.yml`? This will help when multiple files/tables have the same column names.